xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -551,18 +551,14 @@ require(['scriptaculous/effects'], function() {
 }
 
 .applicationPanelMoreButton {
-  padding: 10px 15px 10px 0;
+  padding: 10px 0 0 0;
   border: 0;
   color: $theme.linkColor;
   box-shadow: none;
   background-color: transparent;
   width: 100%;
   display: flex;
-}
-
-.applicationPanelMoreButton:hover,
-.applicationPanelMoreButton:focus {
-  background-color: $theme.highlightColor;
```

Comment:
Here is a demo without focus or hover styles: https://up1.xwikisas.com/#VquQmLMqMHKMMTPBfCwugw
I agree that we should at least have an indicator on hover (on focus, I guess the blue border could be enough).
I propose to only highlight the button on hover. Not on focus, so that it doesn't stay grey after being clicked. 
```
.applicationPanelMoreButton:hover {
  background-color: $theme.highlightColor;
}
```
It's a middle ground that sounds correct to me. Here is a demo with just the 'hover' change of background (and a small border radius to make it look less raw): https://up1.xwikisas.com/#QgoJqFfsWK6PEqc-Xh_5cw

What do you think of this @manuelleduc ?